c/tests/stress.c

```diff
@@ -168,10 +168,10 @@ int main(int argc, char * argv[])
 
   // Call the solver
   prima_result_t result;
-  rc = prima_minimize(algorithm, problem, options, &result);
+  const prima_rc_t rc2 = prima_minimize(algorithm, problem, options, &result);
```

Comment:
This needed to be renamed because there is already an `int rc` in this program.

Reply:
Could you implement the following as a function to avoid the conflict of `rc`? We should **never** use names like `rc2`. 

```c
  char buf[10] = {0};
  time_t t = time(NULL);
  struct tm *tmp = localtime(&t);
  int rc = strftime(buf, 10, "%y%W", tmp);
  if (!rc)
    return 1;
  unsigned seed = atoi(buf);
  printf("Random seed = %d\n", seed);
```

Thanks.

Reply:
I could make it a function, but then what if strftime fails? I would need to have the function return a return code and now we have the same problem. Another option is to add scope brackets around rc. This is legal in C, for example:

```c
int main(int argc, char * argv[]) {
    {
        int rc = 5;
    }
    int rc = 6;
    return rc; // returns 0;
}
```

Or we could name it `strftime_rc`. What do you think?

Reply:
> what if strftime fails? I would need to have the function return a return code and now we have the same problem.

I thought about it. What about setting the seed to some default number, e.g., `42`, `31415`?

Reply:
I kind of like the idea of rotating the random seed on a weekly basis. I checked the docs for strftime and I noticed the it returns the number of bytes written to `buf` (unless that number exceeds the max, in which case it returns 0), so how about we rename the first `rc` to `buf_size`?

Reply:
Yes for the renaming, but it still should be a standalone function. The "FIXME" has been there for a while.

Reply:
I see, that wasn't clear at first, but I don't think it's a good idea. If strftime failed and we return some default failure then we wouldn't know if it failed. We could add a print statement and check logs, but I'd rather it fall 'loudly' as opposed to quietly.

If we make it a standalone function, then the renaming doesn't help since it would be inside the function. Do you have another renaming suggestion? We could make the second one `prima_rc`.

The FIXME is really out of scope for this PR, let's try to keep comments limited to this PR. I believe that more, smaller PRs are better then fewer larger PRs. They're easier to review and therefore quicker to merge, giving us more time to work on other issues.

Reply:
> If strftime failed and we return some default failure then we wouldn't know if it failed.

We decisively disagree. 

Why do we have to know this? The whole purpose of this function is to return an integer that is updated periodically as long as it is possible. I have no interest to know whether strtime fails or not --- surely, a message can be printed if it does fai, which I am sure will never happen.

For reproducibility, the seed has to be printed to the screen. This is because we don't know the exact time when the code is executed not because strtime may fail (which will not happen).

Reply:
Changes made